SplunkAppForWazuh/appserver/static/css/styles/variables.css

```diff
@@ -14,4 +14,8 @@
   --danger-color: #da2062;
   --info-color: #0b3b98;
   --bg-menu-select-color: rgb(0, 68, 90);
+  --bg-active-color: #46BFBD;
+  --bg-pending-color: #FFD35E;
+  --bg-disconnected-color: #F7464A;
+  --bg-never-connected-color: #949FB1;
```

Comment:
**nitpick**: Is the `bg` in the name referring to `background` or it could be misinterpreted?

Reply:
> **nitpick**: Is the `bg` in the name referring to `background` or it could be misinterpreted?

Yes, it's referring to visualization background color of each status.